websockets/deploy.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ -z "$PROJECT" ]; then
+    echo "No PROJECT variable set"
+    exit
+fi
+
+if [ -z "$APIGEE_ENV" ]; then
+    echo "No APIGEE_ENV variable set"
+    exit
+fi
+
+if [ -z "$APIGEE_HOST" ]; then
+    echo "No APIGEE_HOST variable set"
+    exit
+fi
+
+if [ -z "$CLOUD_RUN_SERVICE_URL" ]; then
+    echo "No CLOUD_RUN_SERVICE_URL variable set"
+    exit
+fi
+
+
+
+TOKEN=$(gcloud auth print-access-token)
+
+echo "Installing dependencies"
+npm install
+
+echo "Installing apigeecli"
+curl -s https://raw.githubusercontent.com/apigee/apigeecli/master/downloadLatest.sh | bash
+export PATH=$PATH:$HOME/.apigeecli/bin
+
+echo "Replacing target server URL for $CLOUD_RUN_SERVICE_URL..."
+
+sed -i '' "s/websockets-echo-server.a.run.app/$CLOUD_RUN_SERVICE_URL/" ./apiproxy/targets/default.xml
```

Comment:
Prefer using Target server 
You will need to add the logic for configuring the Target server